### PR TITLE
[v2, windows] Fix dpi scaling for factors other than 100%, 200%,...

### LINKED ISSUE #1097: [v2, windows] DPI Scaling is incorrect for scaling factors other than 100%, 200%, ...
**Description**
DPI Scaling is not applied correctly if the scaling factor is not equal to `n * 100%` where `n` is an integer.

**To Reproduce**
Create a new v2 template. Compile the app for Windows and start it on a windows machine with a scaling of 125%, 150% or 175%. The window is too small and doesn't correctly size itself.

**Expected behaviour**
The scaling should be correct

**Additional context**
The problem is how the scaling of `scaleWithWindowDPI` is implemented in winc here:
https://github.com/leaanthony/winc/blob/fdd739b32c58704d1ff6ff6b44a7e9248b4712fb/controlbase.go#L488-L492

For a scaling factor of 175% the `DPIScaleX` is 1.75 but converting it to an int results in only scaling with 1. Ive provided a PR leaanthony/winc#13 which will fix this issue.

**System Details**
Wails Version: v2.0.0-beta.29
MacOS Version: 12.1.0
Go Version: go1.17.6
Windows Version: Windows 11 21H2

### PR DESCRIPTION
**Pull request by stffabi**
Fix #1097

### CODE REVIEW

File: v2/go.mod [hunk 1]
```diff
@@ -80,3 +80,5 @@ require (
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 )
+
+replace github.com/leaanthony/winc v0.0.0-20220117090042-fdd739b32c58 => github.com/stffabi/winc v0.0.0-20220119085214-13938b57ebc9
```
Comment (stffabi) [inline v2/go.mod:84]: Will be removed as soon as leaanthony/winc#13 has been merged.

**Comment by leaanthony** [comment]
We're good to go 👍👍👍

**Comment by stffabi** [comment]
Thanks for merging, I've updated go.mod.

**Comment by leaanthony** [comment]
Awesome job as always. I better look at tagging a new release

**Comment by stffabi** [comment]
Thanks @leaanthony, as always you're welcome. Maybe you could wait another day for tagging a new release? I've another PR in the pipeline to fix some annoying random panics during WebResourceRequest handling in `go-webview2`. The panics could also occur in several other places too.

**Comment by leaanthony** [comment]
Thanks for the heads up. It's going to be a hell of a release at this rate!

**Pull request merged**